suite-native/device-manager/src/components/DeviceItem/DeviceItemIcon.tsx

```diff
@@ -12,7 +12,7 @@ import { DeviceModelIcon } from '../DeviceModelIcon';
 
 type DeviceItemIconProps = {
     deviceId: TrezorDevice['id'];
-    iconSize: IconSize;
+    iconSize: IconSize | number;
 };
 
 export const DeviceItemIcon = ({ deviceId, iconSize }: DeviceItemIconProps) => {
```

Comment:
nit: What about setting the size as default, so if the component is used elsewhere, we don't have to always set the size via prop?
```suggestion
export const DeviceItemIcon = ({ deviceId, iconSize = 28 }: DeviceItemIconProps) => {
```

Reply:
changed in f72721d0be

Reply:
I was ok with leaving the prop there, just assigning a default to it. But this is also good. When we need to adjust size later for different use cases, we can add the prop later.

Reply:
Yeah. That was exactly my thinking when I made it default first. The prop was obsolete after other changes in previous PR